### PR TITLE
Renderer e2e

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

File: e2e/renderer/e2e/ngif.e2e-spec.ts [hunk 1]
```diff
@@ -2,93 +2,88 @@ import {
...
-        let toggle: ExtendedUIElement;
+        let firstButton: UIElement;
+        let secondButton: UIElement;
+        let conditionalLabel: UIElement;
+        let toggle: UIElement;
 
         before(async () => {
             driver = await createDriver();
-            driverWrapper = new DriverWrapper(driver);
+<<<<<<< HEAD
```
Comment (danielgek) [inline e2e/renderer/e2e/ngif.e2e-spec.ts]: @SvetoslavTsenov merge artifacts over here ;)

Comment (SvetoslavTsenov) [reply e2e/renderer/e2e/ngif.e2e-spec.ts]: @danielgek . Thaks wrong commit :) but already fixed.

**Comment by SvetoslavTsenov** [comment]
e2e

File: e2e/renderer/e2e/ngfor.e2e-spec.ts [hunk 2]
```diff
@@ -2,45 +2,39 @@ import {
...
 
     before(async () => {
         driver = await createDriver();
-        driverWrapper = new DriverWrapper(driver);
-    });
-
-    after(async () => {
-        await driver.quit();
-        console.log("Driver quits!");
+        await driver.driver.resetApp();
```
Comment (sis0k0) [inline e2e/renderer/e2e/ngfor.e2e-spec.ts]: Can we add `resetApp` method to the `AppiumDriver` class that internally invokes `_driver.resetApp()`? (this is a change in the appium plugin)

File: e2e/renderer/e2e/tsconfig.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../tsconfig",
     "compilerOptions": {
+        "sourceMap": true,
```
Comment (sis0k0) [inline e2e/renderer/e2e/tsconfig.json]: why?

File: e2e/renderer/package.json [hunk 4]
```diff
@@ -45,6 +45,7 @@
     "typescript": "~2.2.1"
   },
   "scripts": {
-    "e2e": "tsc -p e2e && mocha --opts ./e2e/config/mocha.opts"
+    "e2e": "tsc -p e2e && mocha --opts ./e2e/config/mocha.opts",
```
Comment (sis0k0) [inline e2e/renderer/package.json]: use the command below

File: e2e/renderer/e2e/setup.ts [hunk 5]
```diff
@@ -1,9 +1,19 @@
-import { startServer, stopServer } from "nativescript-dev-appium";
+import { startServer, stopServer, createDriver, AppiumDriver } from "nativescript-dev-appium";
+
+let driver: AppiumDriver;
```
Comment (sis0k0) [inline e2e/renderer/e2e/setup.ts]: What is this driver for? We are creating a separate driver in each spec file.

**Comment by SvetoslavTsenov** [comment]
e2e

**Comment by SvetoslavTsenov** [comment]
e2e

**Comment by SvetoslavTsenov** [comment]
e2e

**Comment by vakrilov** [comment]
e2e

**Comment by SvetoslavTsenov** [comment]
e2e

**Comment by SvetoslavTsenov** [comment]
e2e

**Pull request merged**